docs/sandstorm_readthedocs_modified_theme/base.html

```diff
@@ -104,6 +104,8 @@
           </div>
 	  {%- block footer %}
           {% include "footer.html" %}
+          <div id="get-help">Need help?<a href="https://groups.google.com/forum/#!forum/sandstorm-dev" class="btn btn-neutral">Sandstorm-dev Group</a> <a href="https://kiwiirc.com/client/irc.freenode.net/?channel=#sandstorm" class="btn btn-neutral">IRC</a>
++         </div>
```

Comment:
Oops, there's still a stray `+` here which shows up on the final site as a "+" to the right of the buttons.